express-routes/server.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log('The date is', new Date());
+
+  app.get('/', (req, res) => {
```

Comment:
These `app.Xxx` calls need to happen _outside_ any `app.use`. They need to be at the top level of the file.